drafts/architecture/components/pulp.adoc

```diff
@@ -0,0 +1,71 @@
+= Pulp 3
+
+== Installation layout
+
+Pulpcore doesn't mandate a specific layout so this module creates and manages this.
+There are some constraints, mostly due to SELinux support.
+
+As part of the installation, it creates a user (default `pulp`) and group (default `pulp`).
+This user gets a home directory (default `/var/lib/pulp`).
+There is also a config dir (default `/etc/pulp`) under which a `settings.py` file is created.
+
+The media root (default `/var/lib/pulp/media`) refers to the https://docs.djangoproject.com/en/2.2/ref/settings/#media-root[MEDIA_ROOT setting].
+In Pulp this should not be served by Apache.
+Instead of https://docs.djangoproject.com/en/2.2/ref/settings/#media-url[MEDIA_URL] Pulp has a dedicated `pulpcore-content` service which can also perform permission checks.
+Only the Pulp services need to read the files so directory permissions are set to `0750`.
+Note this default differs from https://docs.pulpproject.org/settings.html#media-root[Pulp's default].
+A subdirectory of the home directory allows a stricter lockdown and avoids any risk of uploading media files into the wrong directory.
+
+There are also the https://docs.djangoproject.com/en/2.2/ref/settings/#std:setting-STATIC_ROOT[STATIC_ROOT] and https://docs.djangoproject.com/en/2.2/ref/settings/#static-url[STATIC_URL] settings.
+These serve the static assets used by Pulp.
+This includes CSS and Javascript for the HTML pages.
+They're not needed for the application to function, but make browsing the API more convenient.
```

Comment:
I've also noticed that if the assets are not collected, the application refuses to start up. So they must be generated, they should be served (RFC2119 usage of must/should).